src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -545,6 +545,54 @@ async Task<ArraySegment<byte>> WriteLiteralExpressionAsIndex(DotnetObjectId root
             }
         }
 
+        private static bool CheckParametersCompatibility(ElementType? typeCode, JObject value)
+        {
+            if (!typeCode.HasValue)
+                return true;
+            var parameterType = value["type"]?.Value<string>();
+            var parameterClassName = value["className"]?.Value<string>();
+
+            switch (typeCode.Value)
```

Comment:
Do IIUC that we are checking if `value` can be passed as an argument to a method with parameter type `typeCode`?

Reply:
yes.